experimental/checkpoint-restore.md

```diff
@@ -48,7 +48,8 @@ The options for checkpoint create:
 
     Create a checkpoint from a running container
 
-      --leave-running=false    leave the container running after checkpoint
+      --leave-running=false    Leave the container running after checkpoint
+	  --checkpoint-dir         Use a custom checkpoint storage directory
```

Comment:
@AkihiroSuda  I copy it  from the code,  but it works good  as fellows:  https://github.com/yuexiao-wang/docker/blob/update-checkpoint/experimental/checkpoint-restore.md   
What do you have suggestion?